### PR TITLE
fix command line processing error

### PR DESCRIPTION
**Pull request by liufengyun**
Fix command line error for `-help` and `-no-bootcp`.

### CODE REVIEW

File: src/dotty/tools/dotc/config/ScalaSettings.scala [hunk 1]
```diff
@@ -40,7 +40,7 @@ class ScalaSettings extends Settings.SettingGroup {
   val jvmargs = PrefixSetting("-J<flag>", "-J", "Pass <flag> directly to the runtime system.")
   val defines = PrefixSetting("-Dproperty=value", "-D", "Pass -Dproperty=value directly to the runtime system.")
   val toolcp = PathSetting("-toolcp", "Add to the runner classpath.", "")
-  val nobootcp = BooleanSetting("-nobootcp", "Do not use the boot classpath for the scala jars.")
```
Comment (DarkDimius) [inline src/dotty/tools/dotc/config/ScalaSettings.scala]: I do not think we should remain options inherited from scalac, unless they have different meaning.

Comment (liufengyun) [inline src/dotty/tools/dotc/config/ScalaSettings.scala]: OK, then I  change `./bin/dotc` to accept `-nobootcp` instead of `-no-bootcp`.

**Comment by liufengyun** [comment]
I've updated the commit with only changes to `./bin/dotc`.

**Comment by odersky** [comment]
@DarkDimius Can we merge this?

**Comment by DarkDimius** [comment]
LGTM

**Pull request merged**